src/sentry/sentry_metrics/indexer/postgres/models.py

```diff
@@ -15,7 +15,7 @@
 from typing import Mapping, Type
 
 
-@control_silo_model
+@control_silo_with_replication_model
```

Comment:
region silo here too.

Reply:
Yup.  To be clear, this PR's sole focus is getting the orm decorators 1. renamed and 2. adding the new decorator focused on the replicated vs non replicated case.

I am not, focused on getting the decorators in the correct state.  That would be this story: https://getsentry.atlassian.net/browse/HC-243

Reply:
Also, given that this is a decorator rename, in order to support https://github.com/getsentry/getsentry/pull/8577 can I, merge these two within short time when both are green and good, or is it required that I only merge this PR in with a backwards compatible naming, and merge an additional PR removing that rename after the getsentry pr?